desktop/flipper-plugin/src/ui/data-table/DataTable.tsx

```diff
@@ -51,7 +51,7 @@ import {Formatter} from '../DataFormatter';
 import {usePluginInstanceMaybe} from '../../plugin/PluginContext';
 import {debounce} from 'lodash';
 import {useInUnitTest} from '../../utils/useInUnitTest';
-import {createDataSource} from 'flipper-plugin/src/state/createDataSource';
+import {createDataSource} from '../../state/createDataSource';
```

Comment:
Wow, thanks for finding this! IIRC that used in itself to already break the build (correctly), but apparently not anymore